src/index.ts

```diff
@@ -1,6 +1,7 @@
 import * as configureKey from './lib/configureKey';
-import * as exposes from './lib/exposes';
+import {options, presets} from './lib/exposes';
```

Comment:
I would prefer to keep the old as `options` and `presets` are generic, it's not clear they come from exposes in the code.

Reply:
Exposes from the definition conflicts with exposes as an import. That is why I changed it. Any suggestions on how to handle this better?

Reply:
`import * as exposesLib from './lib/exposes';`